### DISCUSSION TITLE
Use custom html templates

### DISCUSSION THREAD

**Discussion by inigohidalgo** [datapane/datapane#151] [Help]
Hello,

Is it possible to specify a different template to use when building reports? I am trying to change the date format in the banner that is generated at the bottom from the american MM/DD/YYYY to DD/MM/YYYY which I can do by modifying [this line](https://github.com/datapane/datapane/blob/ed2f2dd523bba2f08c8ceb4f3c61fcc0a5e556a3/src/datapane/resources/local_report/template.html#L40) to include `.toLocaleString('en-GB', { timeZone: 'UTC' })` in the HTML template, but I would like to avoid having to modify the package's source code, since all my changes would be overwritten in successive updates.

Thanks!

**Comment by mands** [comment]
Hi @inigohidalgo ,

Apologies for the delay in replying here. Firstly, we will fix the issue here such that timezones use the user's local timezone. It's possible to style the report itself, but not the bottom bar at the moment. We will have a think about how we could achieve that and get back to you.

Cheers!
